### ISSUE TITLE
Android Mode size() issue when typing an expression on the "height" argument

### ISSUE THREAD

**Issue by diogotito** [processing/processing#2281]
I was making a game for my Android, and while I was beginning to develop it, I wanted to test it on my laptop. So, I called the _size()_ function, passing my smartphone's screen dimensions, like this:

``` java
void setup() {
  size(540, 960 - 300);
  ...
}
```

I wrote "_960 - 300_" for the height, because 960px was too much for my laptop's screen resolution and I didn't wanted to forget the value of my android's screen height.
Of course, when I wanted to test my game on Android Mode, I forgot to replace the "_size(540, 960 - 300);_" with just "_size(540, 960);_" and I just hit the "Run" button - That's what caused the following bug:

I got this compile error:

```
...
-compile:
    [javac] Compiling 3 source files to C:\Users\DIOGOT~1\AppData\Local\Temp\android737091816678147547sketch\bin\classes
    [javac] C:\Users\DIOGOT~1\AppData\Local\Temp\android737091816678147547sketch\src\processing\test\bells_game\Bells_Game.java:225: error: incompatible types
    [javac]   public String sketchRenderer() { return - 300; }
    [javac]                                           ^
    [javac]   required: String
    [javac]   found:    int
    [javac] 1 error

BUILD FAILED
C:\adt-bundle-windows-x86_64-20130522\sdk\tools\ant\build.xml:713: The following error occurred while executing this line:
C:\adt-bundle-windows-x86_64-20130522\sdk\tools\ant\build.xml:727: Compile failed; see the compiler error output for details.
```

I curiously went to _C:\Users\DIOGOT~1\AppData\Local\Temp\android737091816678147547sketch\src\processing\test\bells_game\_ and opened _Bells_Game.java_, and I found that the preprocessor erroneously generated the _sketchHeight()_ function as

``` java
  public int sketchHeight() { return 960; } // Doesn't return 960 - 300!
```

and even generated a _sketchRenderer()_ function like this:

``` java
  public String sketchRenderer() { return - 300; } // WTF!?
```

**In short:** The preprocessor misinterprets expressions like this _(in the case of "960 - 300")_ by treating each operand as a distinct argument passed to the _size()_ function.
